app/test/fixtures/login.fixture.ts

```diff
@@ -0,0 +1,32 @@
+import { metaMaskFixtures } from '@synthetixio/synpress/playwright'
+import basicSetup from 'test/wallet-setup/basic.setup'
+
+export const test = metaMaskFixtures(basicSetup).extend<{
+  login: () => Promise<void>
+}>({
+  login: async ({ page, metamask }, use) => {
+    await use(async () => {
+      const address = await metamask.getAccountAddress()
+      await page.route(`**/api/auth/token`, async (route) => {
+        await route.fulfill({
+          status: 200,
+          contentType: 'application/json',
+          body: JSON.stringify({
+            success: true
+          })
+        })
+      })
+      await page.addInitScript(() => {
+        window.localStorage.setItem('isAuth', 'true')
+        window.localStorage.setItem('name', 'test')
+        window.localStorage.setItem('ownerAddress', address)
+        window.localStorage.setItem('nonce', '41vj7bz5Ow8oT5xaE')
+        window.localStorage.setItem('authToken', '1234567890')
+      })
```

Comment:
This is not a good idea. 

You are setting the value you expected. 

I think you should test the page like you don't know how the page is coded. 
And you know how the user should interact with it. 
Now you create the test like a user who is interacting with the page. 

What you can do is intercept the backend request but let the page do what he have to do himself.